[{"lineNumber": 6, "col_offset": 0, "nodeName": "deprecation_warning", "type": "Callable[[Any, Any, Any], Any]"}, {"lineNumber": 23, "col_offset": 21, "nodeName": "Exception", "type": "Type[Exception]"}, {"lineNumber": 18, "col_offset": 4, "nodeName": "warn", "type": "Callable"}, {"lineNumber": 18, "col_offset": 4, "nodeName": "warnings", "type": "module"}, {"lineNumber": 19, "col_offset": 8, "nodeName": "format", "type": "Callable[..., str]"}, {"lineNumber": 20, "col_offset": 12, "nodeName": "dep_from", "type": "Any"}, {"lineNumber": 20, "col_offset": 22, "nodeName": "__name__", "type": "Any"}, {"lineNumber": 20, "col_offset": 46, "nodeName": "dep_to", "type": "Any"}, {"lineNumber": 27, "col_offset": 8, "nodeName": "__init__", "type": "Callable[..., None]"}, {"lineNumber": 27, "col_offset": 27, "nodeName": "self", "type": "FatalFlowError"}, {"lineNumber": 27, "col_offset": 33, "nodeName": "msg", "type": "Any"}, {"lineNumber": 20, "col_offset": 22, "nodeName": "__class__", "type": "Any"}, {"lineNumber": 27, "col_offset": 8, "nodeName": "Exception", "type": "Type[Exception]"}, {"lineNumber": 20, "col_offset": 22, "nodeName": "obj", "type": "Any"}]